app/controllers/feeds_controller.rb

```diff
@@ -13,4 +13,9 @@ def show
   def repository_params
     { owner: params[:owner], repo: params[:repo] }
   end
+
+  def repository
```

Comment:
Never mind. It doesn't work to have the `helper_method` be a different name as the method.